From e5057bed6636f4ba4ec3d72ed5966e8dcd17200b Mon Sep 17 00:00:00 2001
From: Tao Liu <tao.liu@windriver.com> 
Date: Mon, 15 May 2017 16:46:28 -0500
Subject: [PATCH 1/1] CGTS-6814: syslog occasionally dropping logs Configure
 Configure the journald rate limit to 5000 messages within 30 seconds. 
 This limit is required to support SM managed processes that share the limit.

---
 src/journal/journald.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/journal/journald.conf b/src/journal/journald.conf
index ef86ffe..2e7180e 100644
--- a/src/journal/journald.conf
+++ b/src/journal/journald.conf
@@ -17,8 +17,8 @@ Storage=none
 #Seal=yes
 #SplitMode=uid
 #SyncIntervalSec=5m
-#RateLimitInterval=30s
-#RateLimitBurst=1000
+RateLimitInterval=30s
+RateLimitBurst=5000
 #SystemMaxUse=
 #SystemKeepFree=
 #SystemMaxFileSize=
-- 
1.8.3.1

